### PR TITLE
Branch17nov

### PR DESCRIPTION
**Pull request by mshd**

### CODE REVIEW

File: src/components/Node/Node.js [hunk 1]
```diff
@@ -380,6 +381,9 @@ const ThemedNode = styled.div`
 const ThemedThumbnail = styled.div`
   width: ${({ theme }) => theme.thumbWidth}px;
   height: ${({ theme }) => theme.thumbHeight}px;
+  .imgMore {
+    display: ${({ theme }) => theme.imageCounter};
```
Comment (ogroppo) [inline src/components/Node/Node.js:385]: let's call it `thumbCounterDisplay` plz

File: src/components/Node/Node.js [hunk 2]
```diff
@@ -371,6 +371,7 @@ export default memo(function Node({
 
 const ThemedNode = styled.div`
   display: ${({ theme }) => theme.nodeDisplay};
+  flex-direction: ${({ theme }) => theme.flexDirection};
```
Comment (ogroppo) [inline src/components/Node/Node.js:374]: `flexDirection` => `cardFlexDirection`

File: src/components/Node/Node.js [hunk 3]
```diff
@@ -371,6 +371,7 @@ export default memo(function Node({
 
 const ThemedNode = styled.div`
   display: ${({ theme }) => theme.nodeDisplay};
```
Comment (ogroppo) [inline src/components/Node/Node.js:373]: can we call everything node or card?

**Review by ogroppo — CHANGES_REQUESTED** [review]

File: src/components/Node/Node.js (hunk 1 above)
Comment (mshd) [reply src/components/Node/Node.js:385]: sure, how about _thumbnaillCounter_

Comment (ogroppo) [reply src/components/Node/Node.js:385]: definitely not with 2 `L`s but yeah, if you want you can rename all the thumb to thumbnail, I'm keen to preserve the name of the css prop you're setting 👍

File: src/App.js [hunk 4]
```diff
@@ -14,6 +14,11 @@ import clsx from "clsx";
 import ls from "local-storage";
 import ReactGA from "react-ga";
 import treeLayout from "./lib/getTreeLayout";
+import {
+  COUSINS_SEPARATION,
+  SAME_GROUP_SEPARATION,
+  SIBLING_SPOUSE_SEPARATION,
+} from "./constants/tree";
```
Comment (ogroppo) [inline src/App.js:21]: you're not using these are you?

**Review by ogroppo — APPROVED** [review]
Gooood stuff!

File: src/App.js (hunk 4 above)
Comment (mshd) [reply src/App.js:21]: no

File: src/components/Node/Node.js (hunk 3 above)
Comment (mshd) [reply src/components/Node/Node.js:373]: so should we use "Node" then?, because they files are called that

Comment (ogroppo) [reply src/components/Node/Node.js:373]: yah fine

File: src/lib/formatEntity.js [hunk 5]
```diff
@@ -61,7 +61,7 @@ export default async function formatEntity(
 
   addBirthDate(formattedEntity, languageCode);
   addDeathDate(formattedEntity, languageCode);
-  addIsInfantDeath(formattedEntity);
+  // addIsInfantDeath(formattedEntity);
```
Comment (ogroppo) [inline src/lib/formatEntity.js:64]: why has this been commented out

**Comment by mshd** [comment]
Because it's only for development, and shouldn't be default

**Pull request merged**